indexer/workers/importer.py

```diff
@@ -182,6 +183,18 @@ def import_story(
         url_hash = unique_url_hash(url)
 
         try:
+            # Check if the document already exists by ID in the alias.
+            # We want to avoid indexing duplicates on ILM index rollover
+            search_response = self.elasticsearch_client().search(
+                index=INDEX_NAME_ALIAS,
+                body={
+                    "query": {"bool": {"filter": {"term": {"_id": url_hash}}}},
+                    "size": 0,
+                },
+            )
+            if search_response["hits"]["total"]["value"] > 0:
+                self.incr_stories("dups", url)
+                return None  # mypy explicit return
```

Comment:
```suggestion
            response = self.elasticsearch_client().search(
                index=INDEX_NAME_ALIAS,
                body={
                    "query": {
                        "bool": {
                            "filter": {
                                "term": {
                                    "_id": url_hash,
                                    "_tier": "data_warm",
                                }
                            }
                        }
                    },
                    # don't need the actual documents.
                    "size": 0,
                },
            )
            if response["hits"]["total"]["value"] > 0:
                raise ConflictError()
```

After rollover, we only need to search `data_warm` tier. `create` on the client will still check for duplicates in `data_hot` tier.

Reply:
@kilemensi searching by data tier makes sense, however we concluded to have all three hot nodes(ramos,bradley,woodward) based on [this discussions](https://github.com/mediacloud/story-indexer/issues/205)

Reply:
Cool @thepsalmist. If this was agreed, the the backup/snapshot plan also needs to be updated. Otherwise stories such as #253 and #254 still refer to ideas of cold/warm tiers.

Reply:
Updated #253 @kilemensi

Reply:
>  # don't need the actual documents.

We dont need to return the actual documents,'optimizing' our search query. Query checks for existence